### ISSUE TITLE
Edison support

### ISSUE THREAD

**Issue by laurb9** [laurb9/tiny_scope#12]
Add support for Intel Edison on Arduino breakout board.

**Comment by laurb9** [comment]
Mock define PSTR and F to allow the code to compile.
Adafruit_SSD1306 fails to compile with missing util/delay.h.
Mocking that results in even more errors.

```
~/arduino-1.5.3-Intel.1.0.4/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp: In constructor 'Adafruit_SSD1306::Adafruit_SSD1306(int8_t)':
~/arduino-1.5.3-Intel.1.0.4/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:157:49: error: no matching function for call to 'Adafruit_GFX::Adafruit_GFX(int, int)'
 Adafruit_GFX(SSD1306_LCDWIDTH, SSD1306_LCDHEIGHT) {
                                                 ^
~/arduino-1.5.3-Intel.1.0.4/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:157:49: note: candidates are:
In file included from ~/arduino-1.5.3-Intel.1.0.4/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:29:0:
~/arduino-1.5.3-Intel.1.0.4/libraries/Robot_Control/Adafruit_GFX.h:41:7: note: Adafruit_GFX::Adafruit_GFX()
 class Adafruit_GFX : public Print {
       ^
```

and

```
In file included from ~/arduino-1.5.3-Intel.1.0.4/hardware/arduino/edison/variants/edison_fab_c/variant.h:12:0,
                 from ~/arduino-1.5.3-Intel.1.0.4/hardware/arduino/edison/libraries/Servo/Wire.h:25,
                 from ~/arduino-1.5.3-Intel.1.0.4/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:27:
~/arduino-1.5.3-Intel.1.0.4/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp: In member function 'void Adafruit_SSD1306::begin(uint8_t, uint8_t, bool)':
~/arduino-1.5.3-Intel.1.0.4/hardware/arduino/edison/cores/arduino/wiring_digital.h:105:53: error: 'PinDescription' has no member named 'pPort'
 #define digitalPinToPort(P)  ( g_APinDescription[P].pPort )
                                                     ^
~/arduino-1.5.3-Intel.1.0.4/hardware/arduino/edison/cores/arduino/wiring_digital.h:108:38: note: in definition of macro 'portOutputRegister'
 #define portOutputRegister(port) ( &(port->PIO_ODSR) )
                                      ^
~/arduino-1.5.3-Intel.1.0.4/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:171:38: note: in expansion of macro 'digitalPinToPort'
     csport      = portOutputRegister(digitalPinToPort(cs));
                                      ^
~/arduino-1.5.3-Intel.1.0.4/hardware/arduino/edison/cores/arduino/wiring_digital.h:106:56: error: 'PinDescription' has no member named 'ulPin'
 #define digitalPinToBitMask(P)  ( g_APinDescription[P].ulPin )
                                                        ^
~/arduino-1.5.3-Intel.1.0.4/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:172:19: note: in expansion of macro 'digitalPinToBitMask'
     cspinmask   = digitalPinToBitMask(cs);
                   ^
```

**Comment by laurb9** [comment]
http://hackaday.com/2017/06/19/intel-discontinues-joule-galileo-and-edison-product-lines/
